tracing-subscriber/src/filter/env/mod.rs

```diff
@@ -250,7 +250,9 @@ impl EnvFilter {
         use tracing::level_filters::STATIC_MAX_LEVEL;
         use tracing::Level;
 
-        let directives: Vec<_> = directives.into_iter().collect();
+        let fallback_level = Directive::from(crate::fmt::Subscriber::DEFAULT_MAX_LEVEL);
+
+        let directives: Vec<_> = std::iter::once(fallback_level).chain(directives.into_iter()).collect();
```

Comment:
I don't think we should make this change. This means that _all_ `EnvFilter`s will enable the `INFO` level when constructed, which changes behavior significantly (which is why the tests are currently failing for this PR). I'm backing out this part of the change:
```suggestion
        let directives: Vec<_> = directives.into_iter().collect();
```